components/02-elements/button/_button.scss

```diff
@@ -1,36 +1,37 @@
 $button-height                    : 48px !default;
 $button-width                     : 48px !default;
-$button-padding                   : 0 35px !default;
+$button-padding                   : 0 $padding-base !default;
 $button-margin                    : 0 0 5px 0 !default;
```

Comment:
we should use margin inside component? Each button will need it and we won't get a problem during implementation?

Reply:
yeah, this is legacy margin...I thought it was somehow needed but couldn't explain it. You are so right...removed :)